NewHorizons/Builder/Volumes/RepairVolumeBuilder.cs

```diff
@@ -0,0 +1,52 @@
+using NewHorizons.Builder.Props;
+using NewHorizons.Components.Volumes;
+using NewHorizons.External.Modules.Props;
+using NewHorizons.External.Modules.Volumes.VolumeInfos;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using UnityEngine;
+
+namespace NewHorizons.Builder.Volumes
+{
+    public static class RepairVolumeBuilder
+    {
+        public static NHRepairReceiver Make(GameObject planetGO, Sector sector, RepairVolumeInfo info)
+        {
+            // Repair receivers aren't technically volumes (no OWTriggerVolume) so we don't use the VolumeBuilder
```

Comment:
Neither were VanishVolumes smh my head

We should log errors then if they try to set useShape to true yeah?

Reply:
ShapeBuilder will do that itself with this particular overload so we should be good there